admin/linux/build-appimage.sh

```diff
@@ -91,6 +91,14 @@ export QML_SOURCES_PATHS=${DESKTOP_CLIENT_ROOT}/src/gui
 
 ./linuxdeploy-squashfs-root/AppRun --desktop-file=${DESKTOP_FILE} --icon-file=usr/share/icons/hicolor/512x512/apps/Nextcloud.png --executable=usr/bin/${EXECUTABLE_NAME} --appdir=AppDir --output appimage
 
+# Workaround issue #103 and #7231
+rm -rf ./squashfs-root
+APPIMAGE=$(ls *.AppImage)
+"./${APPIMAGE}" --appimage-extract
+rm "./${APPIMAGE}"
+rm ./squashfs-root/usr/lib/libglib-2.0.so.0
+PATH=./linuxdeployqt-squashfs-root/usr/bin:$PATH appimagetool -n ./squashfs-root "$APPIMAGE"
```

Comment:
this seems to be failing
can you try using a relative or absolute path to appimagetool instead of relying on PATH variable ?

Reply:
Looked for the appimagetool in linuxdeploy-x86_64.AppImage and linuxdeploy-plugin-qt-x86_64.AppImag but it's not contained anymore. 
Before the refactoring of #4584, the PR #4830 used the source https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage which does contain the appimagetool binary, checked it by downloading and extracting.

The refactoring changed the source to https://github.com/linuxdeploy/linuxdeploy-plugin-qt/releases/download/continuous/linuxdeploy-plugin-qt-x86_64.AppImage which does not contain the appimagetool binary anymore.

Therefore the fix downloads the appimagetool as an AppImage.